dashboards/prometheus.yml

```diff
@@ -0,0 +1,30 @@
+global:
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  evaluation_interval: 15s
+alerting:
+  alertmanagers:
+  - scheme: http
+    timeout: 10s
+    api_version: v1
+    static_configs:
+    - targets: []
+scrape_configs:
+- job_name: prometheus
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+  - targets:
+    - localhost:9090
+- job_name: libp2p
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /debug/metrics/prometheus
+  scheme: http
+  static_configs:
+  - targets:
+    - host.docker.internal:5001
```

Comment:
This only works on macos. This will fail on linux

Reply:
How would this work on Linux?

Reply:
there's no easy way unfortunately. You can use https://github.com/qoomon/docker-host but that requires binding the host thing to 0.0.0.0 (or at least the bridge interface).

You can do `network_mode=host`, but then that doesn't work for MacOS environmnets.

Reply:
I opened #2387.